app/decorators/user_decorator.rb

```diff
@@ -113,4 +109,14 @@ def considered_new?
 
     created_at.after?(min_days.days.ago)
   end
+
+  # rubocop:disable Rails/Delegate
+  def display_sponsors
```

Comment:
These methods are to load these 2 `UsersSetting` values into the `UserData` object.

Reply:
Noiceeeee! 😎